pkg/printer/printer.go

```diff
@@ -213,6 +213,7 @@ func GetFlowType(f *pb.Flow) string {
 			l7Protocol = "http"
 		case *pb.Layer7_Dns:
 			l7Protocol = "dns"
+			l7Type += fmt.Sprintf(" %s", l7.GetDns().ObservationSource)
```

Comment:
```suggestion
			l7Type += " " + l7.GetDns().ObservationSource
```

Why not skip the costly `Sprintf()`?